src/software/ai/ai.h

```diff
@@ -24,15 +24,15 @@ class AI final
                 std::shared_ptr<const AIControlConfig> control_config);
 
     /**
-     * Calculates the Primitives that should be run by our Robots given the current
+     * Calculates the PrimitiveSetMsg that should be run by our Robots given the current
      * state of the world.
```

Comment:
I think `Primitives` makes more sense then the type here since this is supposed to be a high level semantic explanation?